Source_Codes/Reset_PIC/ResetMain.c

```diff
@@ -31,16 +31,38 @@ Void TIMER1_ISR()
 void settings()
 {
    //output_low(PIN_C6);
+   // assert(int1 status == 0 || int1 status == 1);

```

Comment:
What does mean `int1`?
If you uncomment this code and compile it, you will get an error and need to fix it.

Reply:
Any type declaration should not be written in assertion, so it should be `assert(status == 0 || status == 1);`.
If you want to declare the type `int1`, write as follows:

```c
int1 status;
assert(status == 0 || status == 1);
```

But it still has an issue that `status` is not defined.